client/lib/signup/flow-controller.ts

```diff
@@ -317,7 +322,10 @@ export default class SignupFlowController {
 			includes( currentSteps, step.stepName )
 		);
 		const pendingSteps = filter( signupProgress, { status: 'pending' } );
-		const completedSteps = filter( signupProgress, { status: 'completed' } );
+		const completedSteps = filter(
+			signupProgress,
+			( step ) => step.status === 'completed' || step.wasSkipped
```

Comment:
Should we need to change this? As the status of skipped step might be `pending`, it will also show in the `pendingSteps`. I feel a little weird 🤔

Reply:
Oh I see what you mean around the naming. I changed it to include skipped steps so that [the check on line 331](https://github.com/Automattic/wp-calypso/blob/trunk/client/lib/signup/flow-controller.ts#L331) passes if the user skipped a step. I've updated the name to be `completedOrSkippedSteps` (it's only used for the one check)

Reply:
Oh I mean the the skipped steps might also show in `pendingSteps`. For example
```
const pendingSteps = [ 'site-options'  ]; // as the step.status === 'pending'
const completedOrSkippedSteps = [ 'site-options' ]; // as the step.wasSkipped === true
```

Reply:
ok, I updated pendingSteps to only include steps that were did not skipped. The array is used to process steps later in this function.

Reply:
👍